api/src/main/scala/io/phdata/rest/WorkspaceController.scala

```diff
@@ -198,7 +199,12 @@ class WorkspaceController[F[_]: Sync: Timer: ContextShift: ConcurrentEffect](
               case e: Throwable =>
                 logger.error(s"Failed to add member to workspace $id: ${e.getLocalizedMessage}", e).pure[F]
             }
-            _ <- emailService.newMemberEmail(id, memberRequest).value
+            emailResult = try {
+              emailService.newMemberEmail(id, memberRequest).value
+            } catch{
+              case e: Exception =>
+                logger.error((s"Failed to send welcoming message, due to this reason: $e"))
```

Comment:
This will still not show part of the exception, use this constructor instead: `logger.error((s"Failed to send welcoming message", e))`

Reply:
@afoerster , updated.